geode-core/src/test/java/org/apache/geode/management/internal/cli/commands/StartMemberUtilsTest.java

```diff
@@ -105,6 +104,31 @@ public void testGeodeOnClasspathIsFirst() {
     assertThat(gemfireClasspath).startsWith(customGeodeCore);
   }
 
+  @Test
+  public void testExtensionsJars() throws IOException {
+
+    // when there is no `extensions` directory
+    String gemfireClasspath = StartMemberUtils.toClasspath(true, new String[0]);
+    assertThat(gemfireClasspath).doesNotContain("extensions");
+
+    if (File.separatorChar == '/') {
+      File extensionsDirectory = new File("/tmp/extensions");
+      try {
+
+        boolean result = extensionsDirectory.mkdir();
+        assertThat(result).isTrue();
+        File jarFile = new File("/tmp/extensions/test.jar");
```

Comment:
This test probably should use TemporaryFile rule, otherwise this will fail on windows?